modules/op-release-notes-1-4.adoc

```diff
@@ -5,29 +5,44 @@
 [id="op-release-notes-1-4_{context}"]
 = Release notes for {pipelines-title} General Availability 1.4
 
-
-[id="new-features-1-4_{context}"]
-== New features
 {pipelines-title} General Availability (GA) 1.4 is now available on {product-title} 4.7.
 
-[IMPORTANT]
+[NOTE]
+====
+In addition to the stable and preview Operator channels, the {pipelines-title} Operator 1.4.0 comes with the ocp-4.6 and ocp-4.5 deprecated channels. These deprecated channels and support for them will be removed in the following release of {pipelines-title}.
```

Comment:
1. I see also `ocp-4.4` on my 4.7 cluster
2. do we explain anywhere what's the difference between `stable` and `preview` channel?

Reply:
Agreed on the diff between `stable` and `preview`, we should probably write something about it.
`stable` is GA and supported, `preview` is where release candidate will be shipped, and thus is not supported.
Or something like that.
cc @siamaksade

Reply:
@ppitonak @vdemeester the implications of Stable and Preview channels are spelled out in step 4C here: https://docs.openshift.com/container-platform/4.7/cicd/pipelines/installing-pipelines.html#op-installing-pipelines-operator-in-web-console_installing-pipelines is that sufficient information or do we want to add any more information there?

Reply:
@ppitonak can you confirm that we want to add ocp-4.4 too?